pkg/apis/kops/cluster.go

```diff
@@ -349,6 +352,18 @@ type ClusterSubnetSpec struct {
 	Type SubnetType `json:"type,omitempty"`
 }
 
+type EgressProxySpec struct {
+	HTTPProxy     HTTPProxySpec `json:"httpProxy,omitempty"`
+	ProxyExcludes string        `json:"excludes,omitempty"`
```

Comment:
We need to figure out how to exclude the internal AWS urls as well, if the cluster is in AWS.